src/main/java/net/minecraftforge/fml/common/versioning/ComparableVersion.java

```diff
@@ -136,16 +161,16 @@ public int compareTo( Item item )
 
             switch ( item.getType() )
             {
-                case INTEGER_ITEM:
+            case INTEGER_ITEM:
                     return value.compareTo( ( (IntegerItem) item ).value );
 
-                case STRING_ITEM:
-                    return 1; // 1.1 > 1-sp
+            case STRING_ITEM:
+                return 1; // 1.1 > 1-sp
 
-                case LIST_ITEM:
-                    return 1; // 1.1 > 1-1
+            case LIST_ITEM:
+                return 1; // 1.1 > 1-1
 
-                default:
+            default:
```

Comment:
...what. Sorry, if I had noticed that, those changes would be long gone. I'm on it.